docs/index.rst

```diff
@@ -22,50 +21,45 @@ and interact with the whole CrateDB cluster in a number of ways.
 Connecting
 ==========
 
-The admin UI is accessible via HTTP at port ``4200``, at a URL like so::
+You can access the admin UI via HTTP on port ``4200``::
```

Comment:
along with your update for this documentation, how would you feel about capitalizing "Admin UI" wherever it appears? (in line with our recent discussion)

Reply:
My understanding of the discussion is that we should stick with "admin UI".  Am I misinterpreting the outcome of that discussion?